.bem/make.js

```diff
@@ -106,8 +106,9 @@ MAKE.decl('SpecNode', {
 
     getLevels : function() {
         return [
-            'libs/bem-core/common.blocks',
-            'common.blocks'
+            'spec.blocks',
+            'common.blocks',
+            'libs/bem-core/common.blocks'
```

Comment:
Order changed to put bem-core code after this library.
`spec.blocks` level added to drop native methods in phantomjs.